### PR TITLE
doc: minor spelling tweaks

### PR DESCRIPTION
**Pull request by josibake**
spelling suggestions as part of #14

### CODE REVIEW

File: 04_wallet.adoc [hunk 1]
```diff
@@ -354,9 +354,9 @@ Once the wallet component is started and any wallets supplied via argument have
 The wallet can be controlled using `bitcoin-cli` or `bitcoin-qt` GUI, and wallet files can be interacted with using the stand-alone `bitcoin-wallet` tool.
 
 Both `bitcoind` and `bitcoin-qt` run a (JSON) RPC server which is ready to service, amongst other things, commands to interact with wallets.
-The command line tool `bitcoin-cli` will allow interaction of any RPC server started by either `bitcoin` or `bitcoin-qt`.
+The command line tool `bitcoin-cli` will allow interaction of any RPC server started by either `bitcoind` or `bitcoin-qt`.
 
-TIP: If using `bitcoin-qt` there is also an RPC console built into the GUI.
+TIP: If using `bitcoin-qt` there is also an RPC console built into the GUI or you can run with `-server=1` to allow access via `bitcoin-cli`.
```
Comment (LarryRuane) [inline 04_wallet.adoc:359]: It seems that `-server` isn't necessary.

File: 04_wallet.adoc [hunk 2]
```diff
@@ -722,7 +722,7 @@ Below is a section of the `AvailableCoins()` function which illustrates availabl
 
 [NOTE]
 ====
-If a `SigningProvider` is found a second check is performed: to see if the coin is "spendable" by calling `IsSolvable()`.
+If a `SigningProvider` is found a second check is performed: to see if the coin is "solvable" by calling `IsSolvable()`.
```
Comment (willcl-ark) [inline 04_wallet.adoc:725]: Nice!

File: 04_wallet.adoc (hunk 1 above)
Comment (josibake) [reply 04_wallet.adoc:359]: hm, perhaps you are referring to running `bitcoind`? the json rpc server is enabled by default when running `bitcoind` (hence no need to explicitly set `-server=1`), but it is disabled by default for `bitcoin-qt`, which means `-server=1` must be set in either `bitcoin.conf` or passed as an argument when starting `bitcoin-qt`, i.e `bitcoin-qt -server=1`

(source: https://github.com/bitcoin/bitcoin/blob/master/doc/JSON-RPC-interface.md#json-rpc-interface)

Comment (LarryRuane) [reply 04_wallet.adoc:359]: Ah, yes, you're right, I was thinking only of `bitcoind`. I was going to suggest possibly making that more explicit, but now that I read it again, I think it's fine (just my bad reading comprehension!).

**Review by LarryRuane — APPROVED** [review]
ACK 6d343a8edb26c06303e043c8474717075118275c

**Review by Ricchkdbanks — APPROVED** [review]

**Pull request merged**